node-check/osgvo-additional-htcondor-config

```diff
@@ -68,7 +68,7 @@ chmod 755 $STASHCP
 if ($STASHCP /osgconnect/public/dweitzel/stashcp/test.file stashcp-test.file) &>/dev/null; then
 
     cat >stashcp_plugin <<EOF
-#!/bin/bash
+#!/bin/sh
```

Comment:
:crying_cat_face:

Reply:
POSIX compliance is the best compliance?

Reply:
This reminds me of when @matyasselmeci would tell us that he had a large book in his office about how to write portable shell.  Then he'd walk into his office and come out with O'Reilly's Programming Perl.